docs/templates/tutorials/distributed-tuning.md

```diff
@@ -0,0 +1,122 @@
+# Distributed Tuning
+
+Keras Tuner makes it easy to perform distributed hyperparameter search. No changes to your code are needed to scale up from running single-threaded locally to running on dozens or hundreds of workers in parallel. Distributed Keras Tuner uses a chief-worker model. The chief runs a service to which the workers report results and query for the hyperparameters to try next. The chief should be run on a single-CPU worker (or alternatively as a separate process on one of the workers).
```

Comment:
"single-CPU worker" -> "single-threaded CPU instance" (to avoid confusion)

Reply:
Done